spec/acceptance/lint_spec.rb

```diff
@@ -62,4 +62,30 @@
       FactoryGirl.lint only_valid_factories
     end.not_to raise_error
   end
+
+  it 'lints traits' do
+    define_model 'User', name: :string do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.